### DISCUSSION TITLE
Template

### DISCUSSION THREAD

**Discussion by EliezerRamirezRuiz** [Rapptz/discord.py#9397] [General] [answered]
can I do a template using this repository?, I don't think and I don't want to get money about this but I'd like to make one for anyone

**Comment by LeoCx1000** [comment]
Template of what?

**Reply by EliezerRamirezRuiz** [reply]
well, I want to make** a template basic about Discord Bot with this library

**Reply by Soheab** [reply]
> well, I want to make** a template basic about Discord Bot with this library



There is already one at [`/examples/basic_bot.py`](https://github.com/Rapptz/discord.py/blob/master/examples/basic_bot.py)

And two commands(?):
- `python -m discord newbot ...`
- `python -m discord newcog ...` 

Source [here](https://github.com/Rapptz/discord.py/blob/master/discord/__main__.py#L235-L342)

**Reply by EliezerRamirezRuiz** [reply]
Well, I didn't know that so thanks anyway but I think you are not understanding me. I will try to explain myself better, I want to make my own template with discord.py and I don't know if there is any problem if I make a template repository.

**Reply by Gobot1234** [reply, marked as answer]
No, there's no problem with doing so

**Reply by EliezerRamirezRuiz** [reply]
thank you for your answer, have a good day.

**Comment by EliezerRamirezRuiz** [comment]
thank you all for the help.
